contrib/roles/linux/kubernetes/tasks/prepare_master.yml

```diff
@@ -221,6 +223,7 @@
 
 - name: Kubernetes Master | Prepare OVN certs
   shell: |
+    set -o errexit
```

Comment:
Same here.

Reply:
Didn't notice the `&&`. I'll be removing `set -o errexit`.